packages/trace-viewer/src/ui/networkTab.tsx

```diff
@@ -26,171 +26,283 @@ import { GridView } from '@web/components/gridView';
+    );
+
+    return { filters };
+  }, [filter]);
 
   if (!networkModel.resources.length)
-    return <PlaceholderPanel text='No network calls' />;
-
-  const grid = <NetworkGridView
-    name='network'
-    items={renderedEntries}
-    selectedItem={selectedEntry}
-    onSelected={item => setSelectedEntry(item)}
-    onHighlighted={item => onEntryHovered(item?.resource)}
-    columns={selectedEntry ? ['name'] : ['name', 'method', 'status', 'contentType', 'duration', 'size', 'start', 'route']}
-    columnTitle={columnTitle}
-    columnWidth={column => column === 'name' ? 200 : 100}
-    render={(item, column) => renderCell(item, column)}
-    sorting={sorting}
-    setSorting={setSorting}
-  />;
-  return <>
-    {!selectedEntry && grid}
-    {selectedEntry && <SplitView sidebarSize={200} sidebarIsFirst={true} orientation='horizontal'>
-      <NetworkResourceDetails resource={selectedEntry.resource} onClose={() => setSelectedEntry(undefined)} />
-      {grid}
-    </SplitView>}
-  </>;
+    return <PlaceholderPanel text="No network calls" />;
+
+  const grid = (
+    <NetworkGridView
+      name="network"
+      items={renderedEntries}
+      selectedItem={selectedEntry}
+      onSelected={item => setSelectedEntry(item)}
+      onHighlighted={item => onEntryHovered(item?.resource)}
+      columns={
+        selectedEntry
+          ? ['name']
+          : [
+            'name',
+            'method',
+            'status',
+            'contentType',
+            'duration',
+            'size',
+            'start',
+            'route',
+          ]
+      }
+      columnTitle={columnTitle}
+      columnWidth={column => (column === 'name' ? 200 : 100)}
+      render={(item, column) => renderCell(item, column)}
+      sorting={sorting}
+      setSorting={setSorting}
+    />
+  );
+  return (
+    <>
+      {!selectedEntry && (
+        <SplitView
+          sidebarSize={30}
+          minSidebarSize={30}
+          sidebarIsFirst={true}
+          orientation="vertical"
+          sidebarHidden={false}
+        >
+          {filters}
+          {grid}
+        </SplitView>
+      )}
+      {selectedEntry && (
+        <SplitView
+          sidebarSize={200}
+          sidebarIsFirst={true}
+          orientation="horizontal"
+        >
+          <NetworkResourceDetails
+            resource={selectedEntry.resource}
+            onClose={() => setSelectedEntry(undefined)}
+          />
+          {grid}
+        </SplitView>
+      )}
+    </>
+  );
 };
 
 const columnTitle = (column: ColumnName) => {
-  if (column === 'name')
-    return 'Name';
-  if (column === 'method')
-    return 'Method';
-  if (column === 'status')
-    return 'Status';
-  if (column === 'contentType')
-    return 'Content Type';
-  if (column === 'duration')
-    return 'Duration';
-  if (column === 'size')
-    return 'Size';
-  if (column === 'start')
-    return 'Start';
-  if (column === 'route')
-    return 'Route';
+  if (column === 'name') return 'Name';
```

Comment:
Just wondering, why has there been chosen for a big `if` list and not for a more generic solution?

Reply:
Those changes are formatting changes, probably leave that part of the file as it is but it clearly needs improvement.